docs/content/faq/how-can-i-use-multiple-clusters-with-cli.md

```diff
@@ -0,0 +1,6 @@
+# How can I use multiple Flynn clusters with the CLI tool
+
+Flynn's CLI multiple clusters out of the box. Use the `-c` option to specify a cluster by name.
+
+    # List apps on a Flynn cluster named production
+    $ flynn -c production apps
```

Comment:
I may be missing something, but that does not work for me. I always have to explicitly specify the cluster when the app/remote is on any cluster other than the default. e.g.:

    $ flynn -c dev -r flynn create test
    [ok]
    $ git push flynn master
    [ok]
    $ flynn ps
    controller: resource not found
    $ flynn -c dev ps
    [works]